src/main/kotlin/no/nav/familie/oppdrag/repository/OppdragLagerRepositoryJdbc.kt

```diff
@@ -153,11 +160,42 @@ class OppdragLagerRepositoryJdbc(
                 UtbetalingsoppdragForKonsistensavstemming(
                     resultSet.getString("fagsak_id"),
                     resultSet.getString("behandling_id"),
-                    objectMapper.readValue(resultSet.getString("utbetalingsoppdrag"))
+                    objectMapper.readValue(resultSet.getString("utbetalingsoppdrag")),
                 )
             }
         }.flatten()
     }
+
+    override fun hentSisteUtbetalingsoppdragForFagsaker(
+        fagsystem: String,
+        fagsakIder: Set<String>,
+    ): List<UtbetalingsoppdragForKonsistensavstemming> {
```

Comment:
Hvorfor trengs denne, lagrer ikke dere utbetalingsoppdraget hos dere? 👀

Reply:
Tanken var at denne skal brukes for å validere at det vi sender til økonomi og det vi har i tilkjent ytelse stemmer overens. Om vi bruker utbetalingsoppdraget vi har hos oss har vi ingen garanti for at det faktisk er sendt til økonomi.